### PR TITLE
op-guide: remove alpha from grpc proxy

### PR DESCRIPTION
**Pull request by heyitsanthony**

### CODE REVIEW

File: Documentation/op-guide/grpc_proxy.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # gRPC proxy
 
-*This is an alpha feature, we are looking for early feedback.*
```
Comment (xiang90) [inline Documentation/op-guide/grpc_proxy.md]: the endpoint still has /alpha prefix, right? also auth does not work on it?

Comment (heyitsanthony) [inline Documentation/op-guide/grpc_proxy.md]: grpc json gateway is v3alpha; grpc proxy goes through integration tests, etc

Comment (xiang90) [inline Documentation/op-guide/grpc_proxy.md]: oh... sorry. i thought this is the json thing.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**